tests/migration/online_migration/zypper_migration.pm

```diff
@@ -165,10 +165,11 @@ sub run {
 
 sub post_fail_hook {
     my $self = shift;
-    $self->SUPER::post_fail_hook;
     $self->select_serial_terminal;
+    upload_logs '/var/log/zypper.log';
```

Comment:
I think move this line after kill zypper will avoid zypper process still write to zypper.log.

Reply:
will try,  but in this case, the log will not change. even we killed zypper. then next zypper command will complain some previous zypper process was running and can't work. cause
it writes the process ID in some files, but I can't find it.